vignettes/internals.Rmd

```diff
@@ -0,0 +1,79 @@
+---
+title: "Fledge internals"
```

Comment:
these are not really internals though. I was putting this outside of the main vignette as it might be too detailed? :thinking:

Reply:
Most users will only need `bump_version()` and `finalize_version()` . Should we account for that in the reference index and in the table?